mlflow/environment_variables.py

```diff
@@ -440,3 +440,15 @@ def get(self):
 MLFLOW_SYSTEM_METRICS_SAMPLES_BEFORE_LOGGING = _EnvironmentVariable(
     "MLFLOW_SYSTEM_METRICS_SAMPLES_BEFORE_LOGGING", int, None
 )
+
+#: Specifies the minimum file size in bytes to use multipart upload when logging artifacts
+#: (default: ``524_288_000``)
+MLFLOW_MULTIPART_UPLOAD_MINIMUM_FILE_SIZE = _EnvironmentVariable(
+    "MLFLOW_MULTIPART_UPLOAD_MINIMUM_FILE_SIZE", int, 524_288_000
```

Comment:
```suggestion
    "MLFLOW_MULTIPART_UPLOAD_MINIMUM_FILE_SIZE", int, 500 * 1024 ** 2
```

Reply:
Can we use the existing constant variables?

https://github.com/mlflow/mlflow/blob/a4db4ee826765f0365944f83124af8840c72f4d8/mlflow/store/artifact/cloud_artifact_repo.py#L28

Reply:
should we make `cloud_artifact_repo` use env var too? i think it makes more sense to make it configurable